homeassistant/components/conversation/models.py

```diff
@@ -27,6 +27,7 @@ class ConversationInput:
     conversation_id: str | None
     device_id: str | None
     language: str
+    agent_id: str | None
```

Comment:
By not adding a default value, we will break all custom components that use `ConversationInput`.